code/controllers/subsystem/timer.dm

```diff
@@ -376,7 +376,7 @@ SUBSYSTEM_DEF(timer)
 	var/spent = 0
 	/// Holds info about this timer, stored from the moment it was created
 	/// Used to create a visible "name" whenever the timer is stringified
-	var/list/timer_info
+	var/alist/timer_info
```

Comment:
this should not be a list, it's being used as a struct not as an assoc list, that is just an accident of construction